### PR TITLE
chore: adapt to use execute methods

### PR DESCRIPTION
**Pull request by boneskull**
- also upgrades chromedriver
- fixes some broken types
- re-add `source-map-support`

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* master
  * **PR #635** <a href="https://app.graphite.dev/github/pr/appium/appium-uiautomator2-driver/635" target="_blank">Graphite</a> 
    * **PR #636** <a href="https://app.graphite.dev/github/pr/appium/appium-uiautomator2-driver/636" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium-uiautomator2-driver/636?utm_source=stack-comment).

**Comment by boneskull** [comment]
@jlipps Remember some time ago I mentioned that applying execute methods after the fact (as in xcuitest) was difficult because `executeMethod()` provides a tuple of arguments instead of an "options object"?  I ran in to the same problem here, of course.  But what I did instead was just... _don't call `executeMethod()`_.  This isn't ideal (due to inconsistency) but it gets us to "documentation" faster, since we can both use `executeMethodMap` _and_ avoid needing to refactor a million method signatures this way.

**Comment by boneskull** [comment]
(this won't pass a build until its target branch's PR passes a build)

File: package.json [hunk 1]
```diff
@@ -64,15 +64,15 @@
...
+    "appium-chromedriver": "^5.5.1",
     "appium-uiautomator2-server": "^5.12.2",
     "asyncbox": "^2.3.1",
     "axios": "^1.x",
     "bluebird": "^3.5.1",
     "css-selector-parser": "^2.2.3",
     "lodash": "^4.17.4",
     "portscanner": "^2.2.0",
-    "source-map-support": "^0.x",
+    "source-map-support": "^0.5.21",
```
Comment (mykola-mokhnach) [inline package.json:75]: by doing this we only allow minor version updates. Do we want that? Also, in the module readme I've seen it is not needed anymore after Node 12, so can we simply get rid of it?

File: lib/commands/mixins.ts [hunk 2]
```diff
@@ -11,6 +17,7 @@ import type {
 import type {EmptyObject} from 'type-fest';
 import {AndroidUiautomator2Driver} from '../driver';
 import type * as types from './types';
+import {Uiautomator2ExecuteMethodMap} from '../execute-method-map';
```
Comment (mykola-mokhnach) [inline lib/commands/mixins.ts:20]: is Uiautomator2ExecuteMethodMap used anywhere?

**Review by jlipps — APPROVED** [review]
I second @mykola-mokhnach's questions but otherwise LGTM! I think it is fine in this case to do what you did and opt out of the standard executemethod call. It's only a convenience function anyway!

File: lib/commands/mixins.ts (hunk 2 above)
Comment (boneskull) [reply lib/commands/mixins.ts:20]: nope

**Pull request merged**